sections/02_intro.md

```diff
@@ -1,97 +1,155 @@
 ## Introduction
 
-### Potential writing prompt
```

Comment:
@XieConnect It's a fun idea, but I agree it might fit better in the Discussion and future opportunities.  Some of the papers don't fit well in that narrative, such as those oriented more toward chemoinformatics.